main.css

```diff
@@ -0,0 +1,46 @@
+/*
+
+orange: #f5c85f
+gray-bg: #EEEEEE
+dark-gray text: #111111
+gray text: #999999
+
+Font: Open Sans
+
+*/
+
+html {
+  font-family: Open Sans;
+}
+
+body {
+  min-height: 100%;
```

Comment:
I'm not sure that this will do what you expected. If you wanted to occupied all the screen height, try ```min-height: 110vh```